code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -102,6 +102,9 @@
 
 	var/caste_luminosity = 0
 
+	/// if fire_immunity is set to be vulnerable, how much will fire damage be multiplied. Defines in xeno.dm
```

Comment:
Either change how the damage calculation works (as it doubles damage at the multiplier being 1 currently per line 82 of .../carbon/xenomorph/life.dm) or change the description here to say it adds the regular damage again times this number.